In [2]:
# Consider the string given below and answer the following questions 
string = "OQYWFClFhFGAvIWYwGKpmZhnJiyzTslSIhSwvOsqJMEphzmifTkyqOMNpnOtXZxmCfgDYqbaBHAUvIWhMnvwZnEMVDvmEfLrDoQnAZgQEgXQVnmSYkfedpAdhrtpOgORpYLRZYGWdhWYuqQssCUXtTzKRDAhpjUheOzUroZNzWFtZOVwIapzUYtbSbjYNErzQ"
print(len(string))

193


####  Calculate the number of characters in the string

-  193
-  200
-  300
-  500


In [4]:
# Type your code here
print(string.count('a'))

2


#### Count the number of occurences of 'a' in the string

-  1
-  2
-  10
-  5

In [ ]:
# Type your code here

#### Identify whether the string starts with 'if'.

-  True
-  False

In [5]:
# Type your code here
result=string.startswith('if')
print(result)

False


#### Return the substring starting at index 63 and ending at index 88  [Both the indexes are inclusive]

-  'OQYWFClFhFGA'
-  'mCfgDYqbaBHAUvIWhMnvwZnEMV'
-  'IWYwGKpmZhnJiyzTslSIhSwv'
-  'JMEphzmifTkyqOMNpnOtXZ'

In [10]:
# Type your code here
newout=string[63:89]
print(newout)

mCfgDYqbaBHAUvIWhMnvwZnEMV


#### Identify the character at index 45

-  a
-  b
-  z
-  x

In [3]:
# Type your co
newout=string[45]
print(newout)

z
